### DISCUSSION TITLE
The std_out problem

### DISCUSSION THREAD

**Discussion by maojinyuan** [glotzerlab/hoomd-blue#1856] [Support] [answered]
Hi there,
I'm new hoomd-blue user in version 4.x.

In v2.x version, I output the information of system by `hoomd.option.set_msg_file(fname)`
I also notice the module `option` has been removed in v4.x. Below is my code. when i use the `message_filename='message.log'` in first line, I didn't find any message output to the 'message.log'. so, am I right?

```
cpu = hoomd.device.CPU(message_filename='message.log')
simulation = hoomd.Simulation(device=cpu, seed=1)
simulation.create_state_from_gsd(filename='lattice.gsd')

integrator = hoomd.md.Integrator(dt=0.01)
cell = hoomd.md.nlist.Cell(buffer=0.3)
lj = hoomd.md.pair.LJ(nlist=cell)
lj.params[('A', 'A')] = dict(epsilon=1, sigma=1)
lj.r_cut[('A', 'A')] = 2.5
integrator.forces.append(lj)
nvt = hoomd.md.methods.Langevin(filter=hoomd.filter.All(), kT=1.0)

integrator.methods.append(nvt)
simulation.operations.integrator = integrator
simulation.run(1000000)
```

**Comment by joaander** [comment, marked as answer]
HOOMD >= 3 produces very few (unusually no) messages by default. You can write messages with [`cpu.notice('message')`](https://hoomd-blue.readthedocs.io/en/v4.8.1/module-hoomd-device.html#hoomd.device.Device.notice). You can also increase the [`notice_level`](https://hoomd-blue.readthedocs.io/en/v4.8.1/module-hoomd-device.html#hoomd.device.Device.notice_level) to see debug logs from HOOMD.
